spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionServiceTests.java

```diff
@@ -492,6 +499,131 @@ public void executeComposedTask() {
 			assertNull(request.getDefinition().getProperties().get("globalstreamkey"));
 		}
 
+		@Test
+		@DirtiesContext
+		public void executeComposedTaskWithAccessToken() {
+			String dsl = "AAA && BBB";
+			initializeSuccessfulRegistry(appRegistry);
+
+			final OAuth2Authentication oAuth2Authentication = mock(OAuth2Authentication.class);
```

Comment:
The common OAuth2Authentication prep and mock setup can be moved into a method and shared across the methods.    I can do this on merge